component/src/main/java/io/siddhi/extension/store/rdbms/RDBMSEventTable.java

```diff
@@ -878,11 +879,11 @@ private void batchProcessSQLUpdates(String sql, List<Map<String, Object>> update
                     throw new ConnectionUnavailableException("Error performing record update operations. " +
                             "Connection is closed for store: '" + tableName + "'", e);
                 } else {
-                    throw new RDBMSTableException("Error performing record update operations for store '"
+                    throw new DatabaseRuntimeException("Error performing record update operations for store '"
```

Comment:
let's add the query when ever we can

Reply:
We cannot get the query at this point, hence it won't be possible